spec/routing/documents_routing_spec.rb

```diff
@@ -13,11 +13,11 @@
     #end
```

Comment:
Changes on this file where not needed and are not related to the PR but since they are changing just a comment I will let it be here :+1: